datadog_lambda/tracing.py

```diff
@@ -90,6 +89,27 @@ def _context_obj_to_headers(obj):
     }
 
 
+def create_dd_dummy_metadata_subsegment(
+    dd_context, trace_context_source, subsegment_metadata_key
+):
+    """
+    Create a Datadog subsegment to pass the Datadog trace context or Lambda function
+    tags into its metadata field, so the X-Ray trace can be converted to a Datadog
+    trace in the Datadog backend with the correct context.
+    """
+    if not dd_context or (
```

Comment:
With the refactoring, `create_dd_dummy_metadata_subsegment` becomes a more general helper function, I don't think `dd_context` is still the right argument name. Perhaps it should be `subsegment_metadata_value`?